arch/x86/core/fatal.c

```diff
@@ -232,17 +232,33 @@ EXC_FUNC_NOCODE(IV_MACHINE_CHECK);
 #define SGX	BIT(15)
 
 #ifdef CONFIG_X86_MMU
-static void dump_entry_flags(u32_t flags)
+static void dump_entry_flags(x86_page_entry_data_t flags)
 {
-	printk("0x%03x %s %s %s\n", flags,
-	       flags & MMU_ENTRY_PRESENT ? "Present" : "Non-present",
-	       flags & MMU_ENTRY_WRITE ? "Writable" : "Read-only",
-	       flags & MMU_ENTRY_USER ? "User" : "Supervisor");
+#ifdef CONFIG_X86_PAE_MODE
+	printk("0x%x%x %s, %s, %s, %s\n", (u32_t)(flags>>32),
+	       (u32_t)(flags),
+	       flags & (x86_page_entry_data_t)MMU_ENTRY_PRESENT ?
+	       "Present" : "Non-present",
+	       flags & (x86_page_entry_data_t)MMU_ENTRY_WRITE ?
+	       "Writable" : "Read-only",
+	       flags & (x86_page_entry_data_t)MMU_ENTRY_USER ?
+	       "User" : "Supervisor",
+	       flags & (x86_page_entry_data_t)MMU_ENTRY_EXECUTE_DISABLE ?
+	       "Execute Disable" : "Execute Enabled");
+#else
+	printk("0x%03x %s, %s, %s\n", flags,
+	       flags & (x86_page_entry_data_t)MMU_ENTRY_PRESENT ?
+	       "Present" : "Non-present",
+	       flags & (x86_page_entry_data_t)maMMU_ENTRY_WRITE ?
```

Comment:
?

Reply:
Strange I don't know how that got in. I did a test on qemu_x86_iamcu just before the commit.
Will fix it. This is causing the CI issues.